[2025-06-06T18:49:36.481+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:33.246134+00:00/task_id=transform_spotify permission to 509
[2025-06-06T18:49:36.482+0000] {logging_mixin.py:188} INFO - Failed to change /opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:33.246134+00:00/task_id=transform_spotify permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:33.246134+00:00/task_id=transform_spotify'
[2025-06-06T18:49:36.499+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:33.246134+00:00/task_id=transform_spotify permission to 509
[2025-06-06T18:49:36.500+0000] {logging_mixin.py:188} INFO - Failed to change /opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:33.246134+00:00/task_id=transform_spotify permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:33.246134+00:00/task_id=transform_spotify'
[2025-06-06T18:49:36.524+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_music_pipeline.transform_spotify manual__2025-06-06T18:49:33.246134+00:00 [queued]>
[2025-06-06T18:49:36.528+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_music_pipeline.transform_spotify manual__2025-06-06T18:49:33.246134+00:00 [queued]>
[2025-06-06T18:49:36.529+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-06T18:49:36.537+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_spotify> on 2025-06-06 18:49:33.246134+00:00
[2025-06-06T18:49:36.541+0000] {standard_task_runner.py:60} INFO - Started process 1068 to run task
[2025-06-06T18:49:36.543+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_music_pipeline', 'transform_spotify', 'manual__2025-06-06T18:49:33.246134+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp3xxxaj6n']
[2025-06-06T18:49:36.544+0000] {standard_task_runner.py:88} INFO - Job 7: Subtask transform_spotify
[2025-06-06T18:49:36.578+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:33.246134+00:00/task_id=transform_spotify permission to 509
[2025-06-06T18:49:36.578+0000] {logging_mixin.py:188} INFO - Failed to change /opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:33.246134+00:00/task_id=transform_spotify permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=etl_music_pipeline/run_id=manual__2025-06-06T18:49:33.246134+00:00/task_id=transform_spotify'
[2025-06-06T18:49:36.579+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_music_pipeline.transform_spotify manual__2025-06-06T18:49:33.246134+00:00 [running]> on host 3dc3cd419f05
[2025-06-06T18:49:36.629+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_music_pipeline' AIRFLOW_CTX_TASK_ID='transform_spotify' AIRFLOW_CTX_EXECUTION_DATE='2025-06-06T18:49:33.246134+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-06T18:49:33.246134+00:00'
[2025-06-06T18:49:36.630+0000] {etl_pipeline.py:25} INFO - [START] Task: transform_spotify
[2025-06-06T18:49:36.630+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 199, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 216, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/etl_pipeline.py", line 26, in wrapper
    result = func(*args, **kwargs)
  File "/opt/airflow/dags/etl_pipeline.py", line 33, in wrapper
    func(*args, **kwargs)
TypeError: transform_spotify() got an unexpected keyword argument 'conf'
[2025-06-06T18:49:36.637+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=etl_music_pipeline, task_id=transform_spotify, execution_date=20250606T184933, start_date=20250606T184936, end_date=20250606T184936
[2025-06-06T18:49:36.645+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 7 for task transform_spotify (transform_spotify() got an unexpected keyword argument 'conf'; 1068)
[2025-06-06T18:49:36.675+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-06-06T18:49:36.692+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
